integration-tests/deployment/ccip/test_assertions.go

```diff
@@ -352,6 +337,41 @@ func ConfirmExecWithSeqNr(
 	}
 }
 
+func ConfirmNoExecConsistentlyWithSeqNr(
+	t *testing.T,
+	source, dest deployment.Chain,
+	offRamp *offramp.OffRamp,
+	expectedSeqNr uint64,
+	timeout time.Duration,
+) {
+	gomega.NewGomegaWithT(t).Consistently(func() bool {
```

Comment:
I think we're moving away from gomega, use `require.Eventually` or `assert.Eventually`